precise/iaf_psc_alpha_canon.cpp

```diff
@@ -169,8 +169,6 @@ nest::iaf_psc_alpha_canon::State_::get( DictionaryDatum& d, const Parameters_& p
   def< double >( d, names::V_m, y3_ + p.E_L_ ); // Membrane potential
   def< double >( d, "y1", y1_ );                // y1 state
   def< double >( d, "y2", y2_ );                // y2 state
```

Comment:
@mhelias Thanks for catching this. But I am wondering if we should not rather drop these two undocumented recordables. They are not recorded by any test or example, and the other models do not offer them or their equivalent (except for `iaf_psc_alpha`, under a proper and long name).
I would suggest to remove them entirely. 

@jougs Do you think that could lead to any trouble with unhappy users?